dev/io.openliberty.checkpoint_fat/publish/servers/checkpointBells/server.xml

```diff
@@ -17,14 +17,19 @@
         <feature>componenttest-1.0</feature>
         <feature>checkpoint-1.0</feature>
         <feature>bells-1.0</feature>
+        <feature>user.feature.checkpoint.bells-1.0</feature>
```

Comment:
I'm not sure why you don't need `usr:` prefix here to enable the user feature:

`<feature>user.feature.checkpoint.bells-1.0</feature>`

Like this other user feature from our tests: https://github.com/OpenLiberty/open-liberty/blob/57e5fb01a3a1f05f9b771a94c5b54e3a5e5fa166/dev/io.openliberty.checkpoint_fat/publish/servers/TestSPIConfig/server.xml#L19